react/components/MinicartIconButton.tsx

```diff
@@ -22,7 +22,9 @@ const MinicartIconButton = () => {
       icon={
         <span
           onMouseEnter={
-            openOnHover ? () => dispatch({ type: 'OPEN_MINICART' }) : undefined
+            openOnHoverBehavior
+              ? () => dispatch({ type: 'OPEN_MINICART' })
+              : undefined
```

Comment:
Same about this, you could create a `handleMouseEnter`. I know that in this case, you would change the functionality of the code, but I think is ok to always have this callback